File: library/src/main/java/io/vithor/yamvpframework/NetworkException.java
package io.vithor.yamvpframework;

/**
 * Created by hazer on 4/20/15.
 */
public class NetworkException extends Exception {

    public NetworkException() {
    }

    public NetworkException(String detailMessage) {
        super(detailMessage);
    }

    public NetworkException(String detailMessage, Throwable throwable) {
        super(detailMessage, throwable);
    }

    public NetworkException(Throwable throwable) {
        super(throwable);
    }

}

File: rest/src/main/java/io/vithor/yamvpframework/rest/NetworkException.kt
package io.vithor.yamvpframework.rest

/**
 * Created by Vithorio Polten on 4/20/15.
 */
class NetworkException : Exception {

    constructor() {
    }

    constructor(detailMessage: String) : super(detailMessage) {
    }

    constructor(detailMessage: String, throwable: Throwable) : super(detailMessage, throwable) {
    }

    constructor(throwable: Throwable) : super(throwable) {
    }

}
